Public/Src/IDE/Generator/IdeGenerator.cs

```diff
@@ -85,6 +85,12 @@ public static void Configure(ConfigurationImpl config, IStartupConfiguration sta
                     solutionNameCandidate = "ideGenerated";
                 }
 
+                // If a new solution generator is used, change the solution name so users can distinguish between the old and new one.
+                if (ideConfiguration.IsNewEnabled)
+                {
+                    solutionNameCandidate += "New";
+                }

```

Comment:
Are we ready to make "new" the standard and instead start labeling the other implementation as "old" instead?

Reply:
I don't think enough people have tried it so we could declare that we can deprecate the old one